### ISSUE TITLE
double-mode themes

### ISSUE THREAD

**Issue by FranklinYu** [mbadolato/iTerm2-Color-Schemes#372]
Latest iTerm2 support double-mode themes, where one theme consists of light and dark variants. iTerm2 comes with 3 of such themes. Attached a “Regular” theme: [theme.tar.gz](https://github.com/mbadolato/iTerm2-Color-Schemes/files/9076042/theme.tar.gz)

We already have quite a few pairs of themes, such as 3024 Day/Night, and Bluloco Dark/Light.

**Comment by uloco** [comment]
I was going to add bluloco for more platforms. So this means I have to use the light and dark separately 
 atm, correct?

**Comment by FranklinYu** [comment]
Yes, that is correct. Once this is implemented, we should be able to get both themes as a “pair” and iTerm2 will do the switching automatically.
